src/backend/distributed/planner/distributed_planner.c

```diff
@@ -751,24 +747,6 @@ InlineCtesAndCreateDistributedPlannedStmt(uint64 planId,
 	/* after inlining, we shouldn't have any inlinable CTEs */
 	Assert(!QueryTreeContainsInlinableCTE(copyOfOriginalQuery));
 
-#if PG_VERSION_NUM < PG_VERSION_12
```

Comment:
I'm glad that we are removing this ultra-hack. `InlineCtesAndCreateDistributedPlannedStmt ` itself is a hack, but this was even worse